bower.json

```diff
@@ -39,9 +38,9 @@
     "*.md"
   ],
   "resolutions": {
-    "d3": "^3.3.13"
+    "d3": "~3.3.13"
   },
   "devDependencies": {
-    "d3": "^3.3.13"
+    "d3": "~3.3.13"
```

Comment:
Again, the ^ is more appropriate, especially since I added features that only work on 3.5.x:

https://github.com/npm/node-semver#caret-ranges-123-025-004